Buildings/Experimental/DHC/Examples/Cooling/DistrictCooling.mo

```diff
@@ -0,0 +1,187 @@
+  Buildings.Experimental.DHC.Loads.Cooling.BuildingTimeSeriesWithETS buiETS[nLoa](
+    filNam=filNam,
+    mBui_flow_nominal=mBui_flow_nominal)
+    "Vectorized time series building load model connected with ETS for cooling."
+    annotation (Placement(transformation(extent={{30,40},{50,60}})));
+  Buildings.Fluid.Sensors.RelativePressure senRelPre(redeclare package Medium = Medium)
+    "Relative pressure drop sensor"
+    annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=270,
+        origin={-8,-36})));
+protected
+  parameter Modelica.Units.SI.SpecificHeatCapacity cp=Medium.specificHeatCapacityCp(
+    Medium.setState_pTX(
+      Medium.p_default,
+      Medium.T_default,
+      Medium.X_default))
+    "Default specific heat capacity of medium";
+equation
+  connect(weaDat.weaBus, pla.weaBus) annotation (Line(
+      points={{-60,30},{-30,30},{-30,-1.13333},{-29.9667,-1.13333}},
+      color={255,204,51},
+      thickness=0.5));
+  connect(on.y, pla.on) annotation (Line(points={{-59,-10},{-50,-10},{-50,-2.6},
+          {-40.7333,-2.6}}, color={255,0,255}));
+  connect(TCHWSupSet.y, pla.TCHWSupSet) annotation (Line(points={{-59,-50},{-48,
+          -50},{-48,-4.73333},{-40.6667,-4.73333}}, color={0,0,127}));
+  connect(senRelPre.p_rel, pla.dpMea) annotation (Line(points={{-17,-36},{-46,
+          -36},{-46,-6.73333},{-40.6667,-6.73333}},
+                                               color={0,0,127}));
+  connect(dis.port_bDisSup, dis.port_aDisRet) annotation (Line(points={{60,-10},
+          {80,-10},{80,-16},{60,-16}}, color={0,127,255}));
+  connect(pla.port_aSerCoo, dis.port_bDisRet) annotation (Line(points={{-40,
+          -11.3333},{-44,-11.3333},{-44,-60},{8,-60},{8,-16},{20,-16}}, color={
+          0,127,255}));
+  connect(dis.port_aDisSup, pla.port_bSerCoo) annotation (Line(points={{20,-10},
+          {0,-10},{0,-11.3333},{-20,-11.3333}}, color={0,127,255}));
+  connect(senRelPre.port_b, dis.port_bDisRet) annotation (Line(points={{-8,-46},
+          {-8,-60},{8,-60},{8,-16},{20,-16}}, color={0,127,255}));
+  connect(senRelPre.port_a, pla.port_bSerCoo) annotation (Line(points={{-8,-26},
+          {-8,-11.3333},{-20,-11.3333}}, color={0,127,255}));
+  connect(dis.ports_bCon, buiETS.port_aSerCoo)
+    annotation (Line(points={{28,0},{0,0},{0,42},{30,42}}, color={0,127,255}));
+  connect(buiETS.port_bSerCoo, dis.ports_aCon) annotation (Line(points={{50,42},
+          {80,42},{80,0},{52,0}}, color={0,127,255}));
+    annotation (Dialog(group="Network"),
+    Icon(
+      coordinateSystem(
+        preserveAspectRatio=false)),
+    Diagram(
+      coordinateSystem(
+      preserveAspectRatio=false)),
+    __Dymola_Commands(
+      file="modelica://Buildings/Resources/Scripts/Dymola/Experimental/DHC/Examples/Cooling/DistrictCooling.mos" "Simulate and plot"),
+    experiment(
+      StartTime=12960000,
+      StopTime=13564800,
+      Tolerance=1e-06,
+      __Dymola_Algorithm="Dassl"),
```

Comment:
in the `experiment` annotation, please don't include the `Tolerance` and `__Dymola_Algorithm`. You could add these settings in the `.mos` file if it is necessary.

Reply:
Thank you for the suggestion! These settings are alrady in the `.mos` file and I have deleted these two for all the example models.